regular.lua

```diff
@@ -105,29 +122,11 @@ function startQuest()
 end
 
 function result()
-	--Bond exp screen.
-	wait(2)
-	click(QuestResultNextClick)
-
-	--Bond level up screen.
-	if BondRegion:exists(GeneralImagePath .. "bond.png") then
-		wait(1)
-		click(QuestResultNextClick)
-	end
-
-	--Master exp screen.
-	wait(2)
-	click(QuestResultNextClick)
-
-	--Obtained item screen.
-	wait(1.5)
-	click(QuestResultNextClick)
+	
+	-- Click through all of the Result screen items. Does NOT account for Max Bond CE acquisition (yet)
+	continueClick(QuestResultNextClick,35)
 
-	--Extra event item screen.
-	if isEvent == 1 then
-		wait(1.5)
-		click(QuestResultNextClick)
-	end
+	wait(3)

```

Comment:
There should be enough delay in line 127, so I guess lines 129 and 132 can be safely removed.